star_tides/services/databases/mongo/models/guide_model.py

```diff
@@ -0,0 +1,35 @@
+''' star_tides.services.mongo.models.user_model
+'''
+from mongoengine import (
+    Document,
+    StringField,
+    UUIDField,
+    ListField,
+    EmbeddedDocumentField
+)
+
+from star_tides.services.databases.mongo.models.guidance_model import Guidance
+from star_tides.services.databases.mongo.models.project_model import (
+    ProjectModel
+)
+from star_tides.services.databases.mongo.models.contact_model import (
+    ContactModel
+)
+
+
+class Guide(Document):
+    ''' User model
+    '''
+    uuid = UUIDField(binary=False, required=False)
+    name = StringField(required=True)
+    focuses = ListField(required=True)
```

Comment:
I didn't include it in the spec document, but you may want to include an optional `summary` string field for a Guide's introduction. Fine to leave as a TODO for now.

Reply:
I can add that real quick.